src/main/java/frc/robot/commands/IntakeCommands/PickUpFromGroundAndPassToPizzaBox.java

```diff
@@ -31,6 +31,7 @@ public PickUpFromGroundAndPassToPizzaBox(PizzaBoxSubsystem pizzaBoxSubsystem, Ar
         .andThen(new StopNoteContainerMotor(pizzaBoxSubsystem))
         .andThen(new StopIntakeMotors(intakeSubsystem))
         );
+      pizzaBoxSubsystem.hasNote = true;
```

Comment:
When do we want to set `hasNote` to true? Should it be set after one the the sequential commands above?